docs/monitoring/metrics.md

```diff
@@ -335,6 +345,21 @@ Parameters:
 - `ttl` - time-to-live for transmitted UDP packets
 - `addressingMode` - UDP addressing mode to use (UNICAST/MULTICAST)
 
+Example configuration:
+
+{% highlight yaml %}
+
+metrics.reporters: gang
```

Comment:
I would expect that users will not read the full documentation, or even the full reporter section. There will be plenty that jump straight to the reporter of their interest, so we can't rely on people understanding it by reading another part of the docs.

I will keep the names as is.